package.json

```diff
@@ -98,6 +98,6 @@
     "events": "^1.1.1"
   },
   "peerDependencies": {
-    "algoliasearch": ">= 3.1 < 4"
+    "algoliasearch": ">= 3.32.0 < 5"
```

Comment:
Did you notice incompatibilities with `algoliasearch` < 3.32.0?

Reply:
The `algoliasearch-helper-js` is not compatible with versions prior to `3.32.0`.

Reply:
What's broken before 3.32? Narrowing the peer dependency is a major version IIRC